backend/src/yeomanui.ts

```diff
@@ -323,6 +363,10 @@ export class YeomanUI {
   }
 
   private back(partialAnswers: Environment.Adapter.Answers, numOfSteps: number): void {
+    const eventType = EVENT_TYPES.PREVIOUS_STEP;
+    let customEvents = [this.generatorName];
+    getSWA().track(eventType, customEvents);
+    this.logger.trace("SAP Web Analytics tracker was called", {eventType, generatorName: this.generatorName, customEvents});  
```

Comment:
wrap this into a func. can be placed at /swa-tracker/swa-tracker-wrapper.ts or at /swa-tracker/swa-tracker-utils.ts

Reply:
The same as above.